netty/src/main/java/io/grpc/netty/NettyClientHandler.java

```diff
@@ -467,6 +469,25 @@ private void gracefulClose(ChannelHandlerContext ctx, GracefulCloseCommand msg,
     close(ctx, promise);
   }
 
+  private void forcefulClose(final ChannelHandlerContext ctx, final ForcefulCloseCommand msg,
+      ChannelPromise promise) throws Exception {
+    lifecycleManager.notifyShutdown(
+        Status.UNAVAILABLE.withDescription("Channel requested transport to shut down"));
```

Comment:
@nmittler, your statement confuses me. `CANCELLED` does not seem appropriate here, but it could be appropriate for passing into `transportReportStatus`. The "but yeah" also confuses me because I can't tell what it is agreeing with, since I was not advocating being "consistent with this Status".